wp-content/themes/core/components/blocks/core_columns/css/columns.pcss

```diff
@@ -0,0 +1,109 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Core Block: Columns
```

Comment:
Is this all duped from WP Core? It woukd be good to mention in a comment at the top of the file the original location of these styles.